v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/BigTableOutput.java

```diff
@@ -0,0 +1,73 @@
+package com.google.cloud.teleport.v2.transforms.io;
+
+import com.google.bigtable.v2.Mutation;
+import com.google.cloud.teleport.v2.options.ProtegrityDataTokenizationOptions;
+import com.google.protobuf.ByteString;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.apache.beam.sdk.io.gcp.bigtable.BigtableIO;
+import org.apache.beam.sdk.schemas.Schema;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.KV;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.PDone;
+import org.apache.beam.sdk.values.Row;
+import org.apache.commons.lang3.tuple.Pair;
+
+/**
+ * The {@link BigTableOutput} class for writing data from template to BigTable.
+ */
+public class BigTableOutput {
```

Comment:
Let's better call it BigTableIO

Reply:
It will conflict with existing class from SDK
https://beam.apache.org/releases/javadoc/2.25.0/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.html

Reply:
Renamed